docs/download/78.md

```diff
@@ -0,0 +1,176 @@
+    ([ICU-22956](https://unicode-org.atlassian.net/browse/ICU-22956))
+* Time zone data (tzdata) version 2025b (2025-mar).
+
+## ICU4C Specific Changes
+
+* [API changes since ICU4C 77 (Markdown)](https://github.com/unicode-org/icu/blob/maint/maint-78/icu4c/APIChangeReport.md) / [(HTML)](https://htmlpreview.github.io/?https://github.com/unicode-org/icu/blob/maint/maint-78/icu4c/APIChangeReport.html)
+  * TODO: old news here
+  * New APIs for colloquial C++ use of C USet ([ICU-22876](https://unicode-org.atlassian.net/browse/ICU-22876))
+    and C UCollator ([ICU-22879](https://unicode-org.atlassian.net/browse/ICU-22879))
+    * These were added in ICU 76, but some of the new APIs did not actually compile with `U_SHOW_CPLUSPLUS_API=0`.
+      They have been fixed in ICU 77 and thoroughly tested.
+      USetElementIterator now returns std::u16string instead of icu::UnicodeString,
+      and therefore it and related APIs have been changed to `@draft ICU 77`.
+      ([ICU-22954](https://unicode-org.atlassian.net/browse/ICU-22954))
+    * For details about these APIs and an example see the
+      “C++ Header-Only APIs” section of the [ICU 76 Migration Issues](76.md#migration-issues).
+
+## ICU4J Specific Changes
+
+* [API Changes since ICU4J 77](https://htmlpreview.github.io/?https://github.com/unicode-org/icu/blob/maint/maint-78/icu4j/APIChangeReport.html)
+* TODO: old news here
+* The Java technology preview implementation of the CLDR MessageFormat 2.0 specification has been updated to incorporate the
+  [CLDR 46.1 spec](https://www.unicode.org/reports/tr35/tr35-74/tr35-messageFormat.html)
+  plus most but not all of the CLDR 47 changes.
+
+## Known Issues
+
+* TODO: old news here
+* The new MeasureUnit for `portion-per-1e9` works for formatting,
+  but when the new member function `getConstantDenominator()` is called on this unit,
+  it incorrectly returns 0. ([ICU-23045](https://unicode-org.atlassian.net/browse/ICU-23045))
+
+## Migration Issues
+
+* (none yet)
+
+### Migration Issues Related to CLDR
+* See [CLDR 47 migration issues](https://cldr.unicode.org/downloads/cldr-47#migration)
+
+## ICU4C Platform Support
+
+ICU4C requires C++17 and has been tested with up to C++23.
+
+We routinely test on recent versions of Linux, macOS, and Windows.
+
+We accept patches for other platforms.
+
+Windows: The minimum supported version is Windows 7. (See [How To Build And Install On Windows](../userguide/icu4c/build.html#how-to-build-and-install-on-windows) for more details.)
+
+## ICU4J Platform Support
+
+ICU4J works on Java 11..21 (at least).
+
+ICU4J should work on Android API level 21 and later but may require “[library desugaring](https://developer.android.com/studio/write/java8-support#library-desugaring)”.
+
+## Download
+
+### GitHub
+TODO rc tag:
+Source and binary downloads are available on the git/GitHub tag page: <https://github.com/unicode-org/icu/releases/tag/release-78-1>
```

Comment:
I think I've seen download info before. 
So this is maybe duplicated info?

Reply:
same structure as before; summary & quick shortcuts at the top, details & more links later. can revisit.